gnome-extra/cinnamon-session/cinnamon-session-4.8.0.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson gnome2-utils xdg
+
+DESCRIPTION="Cinnamon session manager"
+HOMEPAGE="https://projects.linuxmint.com/cinnamon/"
+SRC_URI="https://github.com/linuxmint/cinnamon-session/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+ FDL-1.1+ LGPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="doc ipv6 systemd"
+
+COMMON_DEPEND="
+	>=dev-libs/glib-2.37.3:2
+	media-libs/libcanberra[pulseaudio]
+	virtual/opengl
+	x11-libs/cairo
+	x11-libs/gdk-pixbuf:2
+	>=x11-libs/gtk+-3:3
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libX11
+	x11-libs/libXau
+	x11-libs/libXcomposite
+	x11-libs/libXext
+	x11-libs/libXrender
+	x11-libs/libXtst
+	x11-libs/pango[X]
+	>=x11-libs/xapps-1.0.4
+
+	systemd? ( >=sys-apps/systemd-183 )
+	!systemd? ( sys-auth/elogind[policykit] )
+"
+
+RDEPEND="${COMMON_DEPEND}
+	>=gnome-extra/cinnamon-desktop-4.8
+"
+DEPEND="${COMMON_DEPEND}"
```

Comment:
Looks like `COMMON_DEPEND` is not needed here. You can first define `DEPEND`, then use that as `RDEPEND`.